x/profiles/spec/01_concepts.md

```diff
@@ -0,0 +1,115 @@
+<!--
+order: 1
+-->
+
+# Concepts 
+
+## Profile
+A Desmos profile is a wrap around a common Cosmos on-chain account that adds some fields to it.
+It is built in order to represent an on-chain social identity that will be unique across the multiple social networks built on top of Desmos. 
+
+Each profile is supposed to identify a single **on-chain identity**, but one human user can have multiple profiles connected each one to a different on-chain address. This is particularly useful if you want to have one private profile, one company profile and so on.
+
+### DTag
+The most important information about a profile it's the `DTag`, which represents the **unique human-readable name** associated to such profile. 
+
+To understand it better, the best reference for this would be Twitter's handles: every user has a unique handle that other users can use to tag them. For example, Jack Dorsey's Twitter handle is `jack` ([@jack](https://twitter.com/jack) on Twitter).
+
+If Jack registered a profile inside Desmos, he would probably like to have the same DTag as his Twitter handle: `jack` (which would be referenced as `@jack` inside Desmos too). 
+
+When creating your profile, make sure you select a DTag that other people can easily remember if you want them to later find you inside Desmos.
+
+Note that you **cannot choose whatever DTag you want**. Instead, you should reference the on-chain parameters to make sure that you respect the rules that have been set. Such rules might contain a set of characters allowed and/or a min/max length the DTag must adhere to. 
```

Comment:
This can't be done, because they are on-chain parameters that can change over time. Adding an example here would require us to periodically check if the specifications are updated with the on-chain data, which is not what we should do